go.sum

```diff
@@ -0,0 +1,100 @@
+github.com/coreos/go-semver v0.3.0 h1:wkHLiw0WNATZnSG7epLsujiMCgPAc9xhjJ4tgnAxmfM=
```

Comment:
I guess xline-client go sdk would be a library. Do we still need to keep go.sum?

Reply:
It seems we need to keep it : https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file

Reply:
Yes, we need to keep it.